modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -86,15 +86,16 @@ If you do not add both channels, you will not be able to create the bootstrap re
 +
 * For {es} 6:
 +
-From the {webui}, add [systemitem]``RHEL6 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES6 x86_64``.
+From the {webui}, add [systemitem]``RHEL Expanded Support 6 x86_64``.
 +
-From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.
+From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem].
```

Comment:
sth went wrong here

Reply:
Line 91 was OK; it is analogous as the Expanded support 7 if we replace 7 -> 6.  That is, the original was fine:

```
From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64``.  
```